hail/src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -71,8 +71,8 @@ class RVD(
   // Exporting
 
   def toRows: RDD[Row] = {
-    val localRowType = rowType
-    map(rv => SafeRow(localRowType.physicalType, rv.region, rv.offset))
+    val localRowType = rowPType
```

Comment:
minor nit, do we need to alias this variable?

Reply:
yes, for serialization reasons.

Reply:
I don't quite understand why; is this garbage collection related? If so, I'm not really sure what the cause is: SafeRow gets a pointer to the physicalType object which should prevent rowType from getting garbage collected before use. I meant to look into this because I noticed a similar issue during FS creation. Could you help me understand?

Reply:
rowPType is a class method on RVD. Calling that method within a closure will serialize the entire RVD into the lambda. This will result in a SerializationError, since RVDs are intentionally not serializable to prevent this kind of mistake from affecting performance.

Binding a local variable means that only that variable is captured in the closure.